src/pages/edge/adobe-journey-optimizer-decisioning/tabs/api-reference.md

```diff
@@ -218,6 +265,91 @@ AEPDecisionScope* decisionScope2 = [[AEPDecisionScope alloc] initWithName: @"myS
 }];
 ```
 
+<Variant platform="ios" api="get-propositions-withTimeout" repeat="12"/>
```

Comment:
Add reference to this in `/adobe-journey-optimizer-decisioning/api-reference.md`

Reply:
done @spoorthipujariadobe